code/modules/surgery/organs/organ_external.dm

```diff
@@ -105,13 +105,12 @@
 /obj/item/organ/external/New(var/mob/living/carbon/holder)
 	..()
 	var/mob/living/carbon/human/H = holder
-	icobase = species.icobase
-	deform = species.deform
+	icobase = dna.species.icobase
+	deform = dna.species.deform
 	if(istype(H))
 		replaced(H)
 		sync_colour_to_human(H)
-	spawn(1)
```

Comment:
@KasparoVy @Crazylemon64 

Any idea why this spawn is here? I removed it, and it didn't seem to have any negative impact, other than resolving runtimes with quickly deleted organs...

Reply:
No ideas off the top of my head, looks to've just been waste from my point of view-- same as below